### PR TITLE
Add GeoIP-GeoLite-data package version 2018.06

### PR DESCRIPTION
**Pull request by Sumynwa**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [ ] Ready to merge

---

###### Summary 
Add package version 2018.06

###### Change Log  
- lint spec
- add entry in cgmanifest
- add entry in licenses

###### Does this affect the toolchain?  
NO

###### Associated issues  
NA

###### Test Methodology

- Pipeline build id:
Buddy Build
[AMD64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=324224)
[ARM64](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=324236)

### CODE REVIEW

File: SPECS-EXTENDED/GeoIP-GeoLite-data/GeoIP-GeoLite-data.spec [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+	if [ -e %{_datadir}/GeoIP/GeoIP.dat.rpmsave ]; then
+		mv %{_datadir}/GeoIP/GeoIP.dat.rpmsave \
+			%{_datadir}/GeoIP/GeoIP.dat
+	else
+		ln -sf GeoLiteCountry.dat %{_datadir}/GeoIP/GeoIP.dat
+	fi
+fi
+exit 0
+
+%files
```
Comment (mfrw) [inline SPECS-EXTENDED/GeoIP-GeoLite-data/GeoIP-GeoLite-data.spec:120]: We are missing the %license macro

Comment (Sumynwa) [reply SPECS-EXTENDED/GeoIP-GeoLite-data/GeoIP-GeoLite-data.spec:123]: Added license

**Review by mfrw — APPROVED** [review]
LGTM :)

File: SPECS-EXTENDED/GeoIP-GeoLite-data/GeoIP-GeoLite-data.spec [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+License:        CC-BY-SA
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://dev.maxmind.com/geoip/legacy/geolite/
+Source0:        %{_mariner_sources_url}/GeoIP.dat.gz
+Source1:        %{_mariner_sources_url}/GeoIPv6.dat.gz
+Source2:        %{_mariner_sources_url}/GeoLiteCity.dat.gz
+Source3:        %{_mariner_sources_url}/GeoLiteCityv6.dat.gz
+Source4:        %{_mariner_sources_url}/GeoIPASNum.dat.gz
+Source5:        %{_mariner_sources_url}/GeoIPASNumv6.dat.gz
```
Comment (PawelWMS) [inline SPECS-EXTENDED/GeoIP-GeoLite-data/GeoIP-GeoLite-data.spec:19]: Where are these source coming from? What are the update steps for a new version of the package?

**Pull request merged**